cmd/helm3.go

```diff
@@ -120,7 +120,26 @@ func (d *diffCmd) template(isUpgrade bool) ([]byte, error) {
 		flags = append(flags, "--set-string", stringValue)
 	}
 	for _, valueFile := range d.valueFiles {
-		flags = append(flags, "--values", valueFile)
+		if strings.TrimSpace(valueFile) == "-" {
+			var bytes []byte
+			var err error
+
+			bytes, err = ioutil.ReadAll(os.Stdin)
+
+			tmpfile, err := ioutil.TempFile("", "stdin-values")
```

Comment:
```suggestion
			tmpfile, err := ioutil.TempFile("", "helm-diff-stdin-values")
```

Perhaps this will better communicate that it's coming from helm-diff?

Reply:
Updated. thanks!